src/plugins/dataflow-tool/components/dataflow-program.tsx

```diff
@@ -139,8 +141,8 @@ export class DataflowProgram extends BaseComponent<IProps, IState> {
             }}
             onDrop={event => {
               event.preventDefault();
-              const nodeType = event.dataTransfer.getData("text/plain");
-              console.log(`Adding ${nodeType}`);
+              // const nodeType = event.dataTransfer.getData("text/plain");
+              // console.log(`Adding ${nodeType}`); I don't think this is implemented
```

Comment:
I don't think it's implemented either. Might be worth just removing it altogether.